scripts/error_codes.py

```diff
@@ -241,7 +241,19 @@ def examine_id_coverage(top_dir, source_id_to_file_names, new_ids_only=False):
         "8312", "8592", "9134", "9609",
     }
 
-    new_source_only_ids = source_only_ids - old_source_only_ids
+    # TODO Cover these with tests and remove from this list as the development of experimental
+    # TODO Solidity progresses. The aim should be to completely get rid of `experimental_only_ids`.
+    experimental_only_ids = {
+        "1017", "1439", "1723", "1741", "1801", "1807", "2015", "2138", "2345", "2399",
+        "2599", "2655", "2658", "2934", "3101", "3111", "3195", "3520", "3570", "3573",
+        "3654", "4316", "4337", "4496", "4504", "4686", "4767", "4873", "4955", "5044",
+        "5094", "5096", "5104", "5195", "5262", "5348", "5360", "5387", "5577", "5714",
+        "5731", "5755", "5904", "6175", "6387", "6388", "6460", "6620", "6739", "6948",
+        "7341", "7428", "7531", "8379", "8809", "8953", "9159", "9173", "9282", "9603",
+        "9658", "9817", "9831", "9988",
```

Comment:
Can you put one per line? It's annoying to rewrap these lists and even if you do not rewrap, the diffs are not very clear when removing multiple codes. It would also make it possible to comment on individual codes if need be.

Reply:
There's 64 of them - you're not seriously suggesting I have them one per line? :D 
Also, `old_source_only_ids` is like this as well.

Reply:
Why not? Just did it and pushed a fixup. Took me 30 sec with a simple editor macro.

And I know that other lists are not in this format yet, which is exactly why I'm requesting this. It's always been an annoyance so I'd rather not use this style for future lists. I added a commit that reformats it too.

Reply:
I wasn't talking about the ease of doing it - but rather about having 60+ lines of error codes.

Reply:
I still don't see the problem. It's not like it's a thousand and even then I'd opt for this format. Keep in mind that we'll be adding/removing codes on this list in almost every PR for a while. Do you like conflicts that much?